src/main/kotlin/me/clip/voteparty/cmds/CommandVoteParty.kt

```diff
@@ -29,16 +30,29 @@ internal class CommandVoteParty(override val plugin: VotePartyPlugin) : BaseComm
 {
 	
 	@Subcommand("addvote")
-	@Syntax("<amount>")
+	@Syntax("<amount> [player]")
 	@Description("Add Vote")
 	@CommandPermission(ADMIN_PERM)
-	fun addVote(issuer: CommandIssuer, @Default("1") amount: Int)
+	fun addVote(issuer: CommandIssuer, @Default("1") amount: Int, @Optional name: String?)
 	{
+		
 		if (amount <= 0)
 		{
 			return sendMessage(issuer, Messages.ERROR__INVALID_NUMBER)
 		}
 		
+		if (!name.isNullOrEmpty())
+		{
+			val user = party.usersHandler[name] ?: return sendMessage(issuer, Messages.ERROR__USER_NOT_FOUND)
+			
+			repeat(amount) {
```

Comment:
this will not trigger parties, should probably just call the event

Reply:
Not entirely sure what you mean by that. I'm not firing the event because they aren't actually voting here unless you think I should treat it like it?

Reply:
Well that's the entire point of adding a vote to a player, the fact that it will emulate an actual vote for them.

Reply:
Alrighty, I've updated that. Hopefully, we're on the same page now.